### ISSUE TITLE
iter_chunks type annotation wrong

### ISSUE THREAD

**Issue by dfroger** [aio-libs/aiohttp#3230]
[read_func](https://github.com/aio-libs/aiohttp/blob/v3.4.1/aiohttp/streams.py#L24) is typed as returning `Awaitable[bytes]`, but it [seems](https://github.com/aio-libs/aiohttp/blob/v3.4.1/aiohttp/streams.py#L42-L43) to be rather `Awaitable[Tuple[bytes, bool]]` (but I don't know the `aiohttp` code so I'm not sure this is the correct type or if it's general enough).

This make the returned type of the public method `iter_chunks` to be wrong.

**Comment by dfroger** [comment]
Note: unpacking the return of `iter_chunks`, typed as `bytes`:

```
async for data, _ in response.content.iter_chunks():
    print(data)
```

[makes strangely ](https://github.com/python/mypy/issues/5547) `data` typed as `int*` and `_` typed as `any`.

**Comment by arthurdarcet** [comment]
@dfroger `iter_chunks` is correctly typed: you should iterate with `async for data in response.content.iter_chunk()`

**Comment by asvetlov** [comment]
Fixed by #3232
